maquetado/scss/tech.scss

```diff
@@ -0,0 +1,64 @@
+$tech-logo-size: 100px;
+
+.technology {
+  align-items: center;
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 320px;
+  margin: 15px;
+  width: 230px;
+  position: relative;
+
+  &::before {
+    border: 1px solid $green;
+    border-radius: 5px;
+    bottom: 10px;
+    content: " ";
+    height: 320px;
+    position: absolute;
+    right: 10px;
+    width: 230px;
+    z-index: -1;
+  }
+}
+
+.logo,
+.technology-title,
+.technology-release,
+.technology-description {
+  margin: 10px 0;
+}
+
+.technologies {
+  display: flex;
+  justify-content: center;
+}
+
+.content-center {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 0 380px;
+}
+
+.logo {
+  height: $tech-logo-size;
+  width: $tech-logo-size;
+}
+
+.technology-title {
+  font-size: $text-medium;
```

Comment:
Now that you know how to join styles with same props, what do you think of adding `font-size` to ```.logo,
.technology-title,
.technology-release,
.technology-description {``` and have it overriden only if necessary? You could even erase this `technology-title`